OpenRA.Mods.Common/Widgets/VideoPlayerWidget.cs

```diff
@@ -46,7 +46,14 @@ public void Load(string filename)
 			if (filename == cachedVideoFileName)
 				return;
 
-			var video = VideoLoader.GetVideo(Game.ModData.DefaultFileSystem.Open(filename), true, Game.ModData.VideoLoaders);
+			var stream = Game.ModData.DefaultFileSystem.Open(filename);
+			var video = VideoLoader.GetVideo(stream, true, Game.ModData.VideoLoaders);
+			if (video == null)
+			{
+				Log.Write("debug", $"Invalid video file {filename} detected.");
+				return;
+			}
```

Comment:
Can we move this into `Open` to cover the videos played via Lua as well?

Reply:
That kills the logging as we have no filename but whatever.

Reply:
Oh right. Hm, if you really think we need the logging we could revert this, as the Lua call seems to be wrapped in a try-catch. But I'm not sure if it's worth it. 🤔